src/notelistview.cpp

```diff
@@ -864,9 +859,9 @@ void NoteListView::onCustomContextMenu(QPoint point)
                 auto action = new QAction(folders[id], this);
                 connect(action, &QAction::triggered, this, [this, id] {
                     auto indexes = selectedIndexes();
-                    for (const auto &index : qAsConst(indexes)) {
-                        if (index.isValid()) {
-                            emit moveNoteRequested(index.data(NoteListModel::NoteID).toInt(), id);
+                    for (const auto &idx : qAsConst(indexes)) {
```

Comment:
This one could be renamed too.

Reply:
Riight. Done! 

P.S.: The formatting on this one looks a bit weird now (to my eyes anyway), because of `clang-format`.